helpers/http_api.js

```diff
@@ -140,14 +140,20 @@ var middleware = {
 		if (!config.api.enabled) {
 			res
 				.status(apiCodes.INTERNAL_SERVER_ERROR)
```

Comment:
status `INTERNAL_SERVER_ERROR`? shouldn't this be `FORBIDDEN`?

Reply:
Good catch, I payed attention to the object change and not the status being returned - test is not new - but this will also be a breaking change as on your previous note wouldn't it? Depending on the answer to the other comment we can change this.

Reply:
I have the same opinion here. Not breaking changing. Just bug fix.
All `AccessDeniedError` should be 403 (`FORBIDDEN`)

Reply:
@lsilvs @ManuGowda I updated the status code and made an small fix in the test to for the code to be validated correctly.